README.md

```diff
@@ -92,24 +92,14 @@ It's easy to restrict your requests using Sequelize's `where` query option. Just
 Team.findOne({ where: { city: 'windsor' }})
 ```
 
-You can also do more complex queries by setting the value of a key to JSON.
```

Comment:
Hi @mdibaiee, sorry for this README.md lines that i've removed before, i thought it was for relationship query example, but i just realized maybe it was for object property method filter, but it's now in README.md again